src/ontology/cl-edit.owl

```diff
@@ -27051,7 +27063,9 @@ SubClassOf(obo:CL_1001109 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000685
 
 # Class: obo:CL_1001111 (kidney loop of Henle thin descending limb epithelial cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:7541952"^^xsd:string) obo:IAO_0000115 obo:CL_1001111 "An epithelial cell that is part of the loop of Henle thin descending limb. It is known in some mammalian species that this cell may express aquaporine-1.")
```

Comment:
From the reference provided:
"The expression in DTL of short nephrons is considerably lower compared with the expression in long nephrons, consistent with the known lower osmotic water permeability of this segment. Furthermore, DTL of short nephrons expressing AQP-1 continue abruptly into a thin limb segment without AQP-1. This suggests the existence of a novel thin limb epithelium in the outer medulla."

Given the above information, consider removing the second sentence from the definition. If you feel compelled, this information could be added as a comment.

Reply:
I agree with you, it is better to remove the second sentence, specially because there is a small segment that does not express AQP-1: 
"This demonstrates the existence of a novel thin limb segment in the outer medulla, which does not express AQP-1 water channels.